src/client/Client.cc

```diff
@@ -380,6 +380,32 @@ vinodeno_t Client::map_faked_ino(ino_t ino)
   return _map_faked_ino(ino);
 }
 
+std::string Client::remove_feature_bits(const std::string& input) {
+  std::vector<int> feature_bits_list;
+  std::string lis;
+  std::stringstream css(input);
+
+  while(getline(css, lis, ',')) {
+    feature_bits_list.push_back(std::stoi(lis));
+  }
+  std::vector<int> final_features_bits_list;
+  for (int val : CEPHFS_FEATURES_CLIENT_SUPPORTED) {
+    if (find(feature_bits_list.begin(), 
+        feature_bits_list.end(), val) == feature_bits_list.end()) {
+      final_features_bits_list.push_back(val);
+    }
+  }
```

Comment:
so basically this code is removing the bits supplied via `input` from `CEPHFS_FEATURES_CLIENT_SUPPORTED` and then returning the resultant string. I think this can be simplified to something like:
```
  std::vector<int> feature_bits_list = CEPHFS_FEATURES_CLIENT_SUPPORTED;
  std::string lis;
  std::stringstream css(input);

  while(getline(css, lis, ',')) {
    int feature = std::stoi(lis);
    feature_bits_list.erase(remove(feature_bits_list.begin(), feature_bits_list.end(), feature), feature_bits_list.end());
  }
```

Reply:
Seems more simplified, updated